ksql-version-metrics-client/pom.xml

```diff
@@ -42,6 +42,7 @@
         <dependency>
             <groupId>com.fasterxml.jackson.core</groupId>
             <artifactId>jackson-databind</artifactId>
+            <version>2.12.3</version>
```

Comment:
Interesting, this didn't have a version before? I wonder if it was somehow just fetching the version (or the dependency itself )as a transitive dependency since many of the other artifacts we pull in also use jackson -- did you test which jackson version was ultimately being pulled in when ksql is compiled?

Reply:
yeah, it was the same that they detected that had a CVE. I don't know enough about mvn to explain that tho

Reply:
Fair enough. Although I guess the flaw with the test I brought up is that we could be pulling in the CVE version just because whatever dependency we're getting jackson from hasn't yet upgraded their version of jackson to fix the CVE. I'm also not an maven expert, but maybe there's a way to detect which dependency is responsible for pulling in this version of jackson?
Who are the maven experts here...I want to say, maybe @agavra or @vvcephei will know?